src/coreclr/interpreter/ee_il_dll.cpp

```diff
@@ -0,0 +1,146 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+#include <stddef.h>
+#include "corjit.h"
+
+#include "interpreter.h"
+#include "ee_il_dll.hpp"
+
+#include <string.h>
+#include <stdio.h>
+
+#ifdef _MSC_VER
+#define DLLEXPORT
```

Comment:
Maybe its CoreClr convention to name things DLL on all platforms, but if that not a naming schema then maybe we could drop the dll naming and use INTERP_API or similar ? Maybe also drop the _dll from the file name if we would like to build this source as a static library in future.

Reply:
This is copy&paste from https://github.com/dotnet/runtime/blob/main/src/coreclr/jit/ee_il_dll.cpp . I agree that the historical dll names, etc. does not make sense. It would be nice to apply the same cleanup both in the regular JIT and the interpreter JIT so that both have the same shape.